src/compose/ComposeBox.js

```diff
@@ -77,8 +63,15 @@ type Props = $ReadOnly<{|
   insets: EdgeInsets,
 
   narrow: Narrow,
-  editMessage: EditMessage | null,
-  completeEditMessage: () => void,
+  isEditing: boolean,
+
+  onSend: (GetState, Dispatch, GetText, string, Narrow) => void,
+
+  defaultMessage?: string,
+  defaultTopic?: string,
```

Comment:
Is "default" an accurate name for these? It feels like "initial" would be more appropriate -- like `initialMessage` and `initialTopic`.

In particular, there isn't a particular expectation that this message value is one that the user will actually go with -- it's a draft and probably incomplete, or the existing content of a message they've decided to edit.

Saying something like "initial" would also highlight that their behavior is somewhat unusual for props, in that they get consulted only when the component is first initialized:
```js
  state = {
    // …
    topic:
      this.props.defaultTopic !== undefined
        ? this.props.defaultTopic
        : /* … */,
    message: this.props.defaultMessage || '',
```

Which is good because that's kind of a gotcha in the interface -- it's the reason that it's important that we pass a `key` that changes when these initial values change. (Or, at the point in the branch where these are introduced: it's not yet a gotcha in the component's interface, but making that the case requires a hack in its implementation, namely the use of `UNSAFE_componentWillReceiveProps`.)

And in fact we don't have the key change in all the cases where these initial values do -- we have it depend on `editMessage` but not `draft`. That gets the right behavior, because we're continuously updating the draft in Redux as the user edits, and the input is uncontrolled so there's no need to change anything to reflect it… but it is odd, so it's good for it to stick out more.